resources/templates/ha-vpn-interop-guide-template/index.md

```diff
@@ -0,0 +1,836 @@
+    --peer-name [PEER_NAME] \
+    --peer-asn [PEER_ASN] \
+    --interface [ROUTER_INTERFACE_NAME_0] \
+    --region [REGION] \
+    
+ The command output should look similar to the following example:   
+ 
+    gcloud compute routers add-bgp-peer router-a \
+    --peer-name peer-b \
+    --peer-asn 65002 \
+    --interface if-tunnel-a-to-on-prem-if-0 \
+    --region us-central1 \
+
+For the second VPN tunnel
+
+1.Add a new BGP interface to the Cloud Router.
+
+    gcloud compute routers add-interface [ROUTER_NAME] \
+    --interface-name [ROUTER_INTERFACE_NAME_1] \
+    --mask-length [MASK_LENGTH] \
+    --vpn-tunnel [TUNNEL_NAME_1] \
+    --region [REGION]
+
+The command output should look similar to the following example:
+
+    gcloud compute routers add-interface router-a \
+    --interface-name if-tunnel-a-to-on-prem-if-1 \
+    --mask-length 30 \
+    --vpn-tunnel tunnel-a-to-on-prem-if-1 \
+    --region us-central1
+    
+    
+1.Add a BGP peer to the interface for the second tunnel.
+
+    gcloud compute routers add-bgp-peer [ROUTER_NAME] \
+    --peer-name [PEER_NAME] \
+    --peer-asn [PEER_ASN] \
+    --interface [ROUTER_INTERFACE_NAME_1] \
+    --region [REGION] \
+
+The command output should look similar to the following example:
+
+    gcloud compute routers add-bgp-peer router-a \
+    --peer-name peer-a \
+    --peer-asn 65002 \
+    --interface if-tunnel-a-to-on-prem-if-1 \
+    --region us-central1 \
+ 
+Verify the Cloud Router configuration
+ 
+    gcloud compute routers get-status router-a \
+     --region us-central1 \
+     --format='flattened(result.bgpPeerStatus[].name,
+       result.bgpPeerStatus[].ipAddress, result.bgpPeerStatus[].peerIpAddress)'
+    
+    result.bgpPeerStatus[0].ipAddress:     169.254.174.249
+    result.bgpPeerStatus[0].name:          peer-a
+    result.bgpPeerStatus[0].peerIpAddress: 169.254.174.250
+    result.bgpPeerStatus[1].ipAddress:     169.254.36.85
+    result.bgpPeerStatus[1].name:          peer-b
+    result.bgpPeerStatus[1].peerIpAddress: 169.254.36.86
+
+    gcloud compute routers describe [ROUTER_NAME] \
+    --region [REGION]
+
+### Configure firewall rules
+
+Configure firewall rules to allow inbound traffic from the on-premises
+network subnets. You must also configure the on-premises network firewall to
+allow inbound traffic from your VPC subnet prefixes.
+
+    gcloud compute firewall-rules create [VPN_RULE] \
+    --network [NETWORK] \
+    --allow tcp,udp,icmp \
+    --source-ranges [IP_ON_PREM_SUBNET]
+    
+The command output should look similar to the following example:    
+    
+     gcloud compute firewall-rules create network-a-to-on-prem \
+    --network network-a \
+    --allow tcp,udp,icmp \
+    --source-ranges 192.168.1.0/24
+    
+## Configure the \<vendor-name>\<vendor product> side
+
+<This section includes sample tasks that describe how to configure the
+on-premises side of the VPN gateway configuration using <vendor-name>
```

Comment:
&lt;vendor-name&gt; is not appearing in the rendered document. I suspect you need to use &lt; and &gt; to make the angle brackets show up.